### PR TITLE
Api endpoints to use Accounting object for reading and writing user properties

### PR DESCRIPTION
**Pull request by saurabhjn76**
Fixes issue #298 
Changes:

-  Added endpoint to write the user properties

- Added  endpoints for reading the user properties
               

Screenshots for the change: 
image
image

the write service accepts two parameter key and value to be stored in User settings
 test locally at [http://127.0.0.1:4000/aaa/changeUserSettings.json?key=theme&value=dark](http://127.0.0.1:4000/aaa/changeUserSettings.json?key=theme&value=dark)
 test locally read service  at [http://127.0.0.1:4000/aaa/listUserSettings.json](http://127.0.0.1:4000/aaa/listUserSettings.json)

### CODE REVIEW

**Comment by saurabhjn76** [comment]
@DravitLochan @rishiraj824 @dynamitechetan @jyoth1raditya @madhavrathi please review

File: src/ai/susi/server/api/aaa/ChangeUserSettings.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+                   accounting.getParent().put("Settings", jsonObject, authorization.getIdentity().isPersistent());
+               }
+               JSONObject result = new JSONObject();
+               result.put("message", "You successfully changed settings to your account!");
+               return new ServiceResponse(result);
+           }
+       }
+
+    }
+}
```
Comment (DravitLochan) [inline src/ai/susi/server/api/aaa/ChangeUserSettings.java]: Please provide EOL here!

File: src/ai/susi/server/api/aaa/ListUserSettings.java [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    public ServiceResponse serviceImpl(Query query, HttpServletResponse response, Authorization authorization, JsonObjectWithDefault permissions) throws APIException {
+        if ( authorization.getIdentity()!=null ) {
+            Accounting accouting = DAO.getAccounting(authorization.getIdentity());
+            return new ServiceResponse(accouting.getParent().toJSON());
+        } else {
+            throw new APIException(400, "Specified User Setting not found, ensure you are logged in");
+        }
+
+    }
+}
```
Comment (DravitLochan) [inline src/ai/susi/server/api/aaa/ListUserSettings.java]: Please provide EOL here!

**Comment by saurabhjn76** [comment]
@DravitLochan  thanks :+1: made the changes .

**Comment by saurabhjn76** [comment]
@isuruAb please review

**Pull request merged**